### PR TITLE
[TMVA][SOFIE] Add Support for Conv Operator in RModelParsers for PyTorch & Keras

### PR DESCRIPTION
**Pull request by sanjibansg**
This PR adds support for parsing Convolution layers from PyTorch and Keras models into an RModel object.

## Progress
- [x] Support for PyTorch Convolution Layers (Conv2D)  
- [x] Support for Keras Convolution Layers(Conv2D)          
  - [x] Support for `padding="valid"`
  - [x] Support for `padding="same"`

## Checklist:
- [ ] tested changes locally

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2004/default with flags -Dtmva-sofie=On
